test/e2e/framework/pod/create.go

```diff
@@ -85,8 +100,19 @@ func CreateSecPod(client clientset.Interface, namespace string, pvclaims []*v1.P
 
 // CreateSecPodWithNodeSelection creates security pod with given claims
 func CreateSecPodWithNodeSelection(client clientset.Interface, namespace string, pvclaims []*v1.PersistentVolumeClaim, inlineVolumeSources []*v1.VolumeSource, isPrivileged bool, command string, hostIPC bool, hostPID bool, seLinuxLabel *v1.SELinuxOptions, fsGroup *int64, node NodeSelection, timeout time.Duration) (*v1.Pod, error) {
```

Comment:
I am thinking to make this method also take in PodInfo as an argument instead of listing them all out here